git-prompt.sh

```diff
@@ -194,6 +198,23 @@ __posh_git_echo () {
         false) ShowStashState=false ;;
         *)     ShowStashState=true ;;
     esac
+    local EnableStatusSymbol=`git config --bool bash.enableStatusSymbol`
+    case "$EnableStatusSymbol" in
+        true)  EnableStatusSymbol=true ;;
+        false) EnableStatusSymbol=false ;;
+        *)     EnableStatusSymbol=true ;;
+    esac
+
+    local branchIdenticalStatusSymbol=''
```

Comment:
Hm... I really should have stuck with snake case here, but I forgot exactly why I chose not to.